jobs/rocoto/gempak.sh

```diff
@@ -5,13 +5,19 @@ source "${HOMEgfs}/ush/preamble.sh"
 ###############################################################
 . "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
 status=$?
-[[ ${status} -ne 0 ]] && exit ${status}
+[[ "${status}" -ne 0 ]] && exit "${status}"
```

Comment:
As long as you are changing it
```suggestion
if (( status != 0 )); then exit "${status}"; fi
```